packages/web-components/fast-foundation/src/radio-group/radio-group.template.ts

```diff
@@ -18,9 +23,9 @@ export function radioGroupTemplate<T extends FASTRadioGroup>(): ElementViewTempl
         >
             <slot name="label"></slot>
             <div
-                class="positioning-region ${x =>
-                    x.orientation === Orientation.horizontal ? "horizontal" : "vertical"}"
+                class="positioning-region ${x => x.orientation}"
                 part="positioning-region"
+                ${ref("positioningRegion")}
```

Comment:
@radium-v Is this needed for the component? I don't see it used but I see it was added here.